hardhat.config.js

```diff
@@ -8,11 +8,13 @@ const bodyParser = require("body-parser");
 const ethers = require("ethers");
 
 const { FORKED_XDAI_CHAINID } = require("./helpers/constants");
+
+require("@nomiclabs/hardhat-ethers");
 require("@nomiclabs/hardhat-truffle5");
+require("@solidstate/hardhat-4byte-uploader");
 require("hardhat-contract-sizer");
 require("hardhat-storage-layout-changes");
-require("@solidstate/hardhat-4byte-uploader");
-require("@nomiclabs/hardhat-ethers");
+process.argv.includes("node") ? undefined : require("hardhat-tracer");
```

Comment:
Would it be possible to include this require only in the `node` task?

Reply:
Isn't that exactly what we don't want to do?

Reply:
Yup, I read the conditional backwards. Nevermind.